### DISCUSSION TITLE
v6306

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#160] [General]
Lots of stuff changed/added.

Some things:

1) Each icon's command can be customized, including custom icons and color icons

2) Mouse events for unloaded tabs which can override the default events

3) Reload Action which does the tab action and reload and Hard Action which is the same but hard reloads

4) Signals can now send the active tab

5) Edits Info command

Also added some new show/tab_list commands like Show Nodes, Show Domain, and Show Title

**Comment by N3C2L** [comment]
Fantastic additions! Thanks a bunch!
